nano/core_test/websocket.cpp

```diff
@@ -160,6 +160,72 @@ TEST (websocket, subscription_edge)
 	node1->stop ();
 }
 
+// Test client subscribing to changes in active_difficulty
+TEST (websocket, subscribe_active_difficulty)
```

Comment:
Suggeset keeping the naming convention of the tests: `websocket.active_difficulty`